### DISCUSSION TITLE
DacFx v160 Public Preview

### DISCUSSION THREAD

**Discussion by dzsquared** [microsoft/DacFx#11] [General]
> The DacFx v160 public preview release is now available as a NuGet package for use in testing or updating existing applications to Microsoft.Data.SqlClient.  External DacFx APIs are not altered by this update, however the dependency on System.Data.SqlClient is removed.  An example application leveraging the v160 release of DacFx is available here.  Existing applications will continue to use the generally available v150 NuGet package unless a major version is not specified and preview releases are included in the package dependencies.  

The v160 preview is currently available only as a NuGet: https://www.nuget.org/packages/Microsoft.SqlServer.DacFx/

Microsoft.Data.SqlClient is a data provider for Microsoft SQL Server and Azure SQL Database for both .NET Framework and .NET Core.  Going forward, support for new SQL Server features will be implemented in Microsoft.Data.SqlClient. More about Microsoft.Data.SqlClient is available at the repository: https://github.com/dotnet/SqlClient
(you may notice several fantastic community members have contributed to SqlClient)

We're happy to discuss incoming changes and answer questions 💬

**Comment by adermon** [comment]
Sorry if this seems a stupid question but where is the source code for Microsoft.SqlServer.DacFx? is this open or closed source?. I came here as its the project site link from  [https://www.nuget.org/packages/Microsoft.SqlServer.DacFx/](https://www.nuget.org/packages/Microsoft.SqlServer.DacFx/) to track an issue. I have posted in the issue #18

**Reply by dzsquared** [reply]
Not at all a stupid question - DacFx is currently closed source but may become open source in the future.

**Reply by adermon** [reply]
Ahh OK thanks, but i guess his is the correct place to post issues relating to it. The issue I referenced has been around for some time, is there anyway to get some traction on it?
